src/main/kotlin/com/hrothwell/mal/util/MalUtil.kt

```diff
@@ -46,19 +32,10 @@ class MalUtil {
     fun openUrl(url: String) {
       if (shouldOpenUrls) {
         try {
-          // use cmd if on windows to just open it
-          if (System.getProperty("os.name").lowercase().contains("windows")) {
-            Runtime.getRuntime().exec(
-              """
-        cmd.exe /c start "" "$url"
-      """.trimIndent()
-            )
+          if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
+            Desktop.getDesktop().browse(URI(url))
```

Comment:
This seems to be the culprit for why the `dll` files are now added. I don't know why, I don't know how, but removing this and replacing it with what was there prior seemed to not generate them when rerunning the bat file.

Reply:
Probably something to do with how this works under the hood